python/lsst/pipe/tasks/diff_matched_tract_catalog.py

```diff
@@ -168,20 +172,25 @@ class DiffMatchedTractCatalogConfig(
     pipeBase.PipelineTaskConfig,
     pipelineConnections=DiffMatchedTractCatalogConnections,
 ):
+    _compute_stats: bool = False
+
```

Comment:
I read your comment on this on Jira, however, I think my preference would be to introduce an immediately deprecated boolean config field, which gets removed at the same time as the other deprecated assets. That's certainly the more standard approach, and I think the path of least potential confusion.